AndroidTestAutomationStarter/kotlinapi/gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off

```

Comment:
So now we can run the whole thing on Windows? 
Or rather- have you tried?

Reply:
Not sure - but the gradlew.bat is always included if you use the cradle wrapper so no I didn't try